src/transformers/models/bloom/tokenization_bloom_fast.py

```diff
@@ -109,7 +99,6 @@ class BloomTokenizerFast(PreTrainedTokenizerFast):
 
     vocab_files_names = VOCAB_FILES_NAMES
     pretrained_vocab_files_map = PRETRAINED_VOCAB_FILES_MAP
-    max_model_input_sizes = PRETRAINED_POSITIONAL_EMBEDDINGS_SIZES
     model_input_names = ["input_ids", "attention_mask"]
     slow_tokenizer_class = None
 
```

Comment:
WDYT of adding a comment here to keep a track in the code of the reason why this tokenizer doesn't have this field fill?
```suggestion
# No `max_model_input_sizes` as BLOOM uses ALiBi positional embeddings
```